components/new-contribution-flow/ContributionFlowSuccess.js

```diff
@@ -64,7 +69,17 @@ ShareLink.defaultProps = {
   target: '_blank',
 };
 
-// only meant until page is hooked up to actual success data
+// GraphQL
+const orderSuccessMemberQuery = gql`
```

Comment:
Should be a GQLV2 query

Reply:
This is just a copy of the current `orderSuccessMemberQuery` v1 for now. I wasn't going to write the updated v2 query until I had access to the completed contribution data.